modules/behavioural_qc/php/behavioural_qc.class.inc

```diff
@@ -489,13 +488,16 @@ class Behavioural_QC extends \NDB_Form
         $site=null,
         $project=null
     ) {
-        $config         =& \NDB_Config::singleton();
+        $config = \NDB_Factory::singleton()->config();
+        $DB     = \NDB_Factory::singleton()->database();
         $ddeInstruments = $config->getSetting('DoubleDataEntryInstruments');
-        for ($i=0; $i<count($ddeInstruments); ++$i) {
+
+        // Why is the first instrument skipped?
```

Comment:
Could we remove this comment as it should start at $ddeInstruments[0]?

Reply:
oops, sorry, I clicked merge before seeing your comment. I'll send a PR to do that and make another change which I think shouldn't have been in this that I didn't want to block over.

Reply:
should have requested change.

Reply:
Sent #5936